src/ProjectDataProvider.ts

```diff
@@ -330,19 +334,20 @@ export module ProjectDataProvider {
         `-u`,
         `github.com/fabric8-analytics/cli-tools/gomanifest`,
         `&&`,
-        Config.getGoExecutable(),
-        `run`,
-        `github.com/fabric8-analytics/cli-tools/gomanifest`,
+        `${goManifestPath}`,
         `"${vscodeRootpath}"`,
         `"${goGraphFilePath}"`,
-        `"${Config.getGoExecutable()}"`
+        `"${Config.getGoExecutable()}"`,
       ].join(' ');
 
       console.log('CMD : ' + cmd);
       outputChannelDep.addMsgOutputChannel('\n CMD :' + cmd);
+      let env = process.env
+      env.GOPATH = goPath;
+      outputChannelDep.addMsgOutputChannel('\n env.GOPATH : ' + env.GOPATH);
```

Comment:
This is incorrect, you are mutating the global process.env. Previously suggested snippet should do the job.

Reply:
My bad, changes are taken in PR.